src/daos/sql-query-builder.php

```diff
@@ -112,15 +126,17 @@ public function or_where( $string ) {
 
   /**
    * Adds a variable to be bound to in the prepared statement
+   *
    * @param $name : string
    * @param $value : object
    */
-  public function bind_value( $name, $value ) {
-    $this->values_to_bind[ $name ] = $value;
+  public function bind_value( $name, $value, $data_type = \PDO::PARAM_STR ) {
```

Comment:
`$data_type` should default to `false` and if its false not get set in the values_to_bind.